Gemfile

```diff
@@ -1,17 +1,2 @@
-# Copyright 2017 JanusGraph Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-source 'https://rubygems.org'
-gem 'jekyll', '~> 3.3'
-gem 'jekyll-theme-slate', '~> 0.0.4'
+source "https://rubygems.org"
+gemspec
```

Comment:
Travis is failing, so I think there's something wrong with your Gemfile.

Reply:
I am not yet sure why it fails. There seems to be a problem with the version. I will try to resolve it tomorrow.

Reply:
Fixed